src/Shared/PropertyHelper/PropertyHelper.cs

```diff
@@ -42,11 +40,11 @@ internal class PropertyHelper
     private static readonly ConcurrentDictionary<Type, PropertyHelper[]> PropertiesCache = new();
 
     private static readonly ConcurrentDictionary<Type, PropertyHelper[]> VisiblePropertiesCache = new();
-
     // We need to be able to check if a type is a 'ref struct' - but we need to be able to compile
     // for platforms where the attribute is not defined, like net46. So we can fetch the attribute
     // by late binding. If the attribute isn't defined, then we assume we won't encounter any
     // 'ref struct' types.
+    [UnconditionalSuppressMessage("Trimming", "IL2026:Members annotated with 'RequiresUnreferencedCodeAttribute' require dynamic access otherwise can break functionality when trimming application code")]
```

Comment:
Why do you need to suppress a warning here. Linker should be able to figure this out - as in it should be able to find the attribute type in the app/closure and not warn on this at all.
If you have a repro where this warns, could you please send it over to me?

Reply:
The analyzer warning me about it and offered this suppression as a codefix. We could look at it at the same time we look at figuring out the other suppressions you asked me not to configure.

Reply:
I would not trust the analyzer on this - just yet. There were lot of issues fixed in the analyzer recently for one thing. (Not saying that you should not pay attention and tells us about any discrepancies). The general rule we're trying hard to follow is:
- analyzer should not report warnings which the linker does not

So in short, if you see analyzer warning, but linker doesn't -> don't trust the analyzer, it's very probably wrong (and let us know so that we can fix it).

Reply:
Change to `private static readonly Type IsByRefLikeAttribute = typeof(System.Runtime.CompilerServices.IsByRefLikeAttribute);`